### ISSUE TITLE
Rest argument can be split from ampersand (&)

### ISSUE THREAD

**Issue by NoahTheDuke** [kkinnear/zprint#252]
Hey there!

Sometimes, the length of a given set of params can be long enough that the rest argument (`& body`, generally) is split across lines. Would it be possible to treat the `&` and the following argument as a single entity, and either split before or after them but never between?

Thanks so much!

```clojure
(defn example
  [vvvvvvvveeeeeeeerrrrrrryyyyy looooooooooooooooooonnnnggggg paraaaaaaaaaams &
   body]
  (+ 1 2 3))
```

**Comment by NoahTheDuke** [comment]
hah No problems, I don't mind waiting. Thank you so much for your tireless work!

**Comment by kkinnear** [comment]
This is now implemented in `1.2.5`.   Use `:vector {:no-wrap-after #{"&"}}` to get the behavior you want!

**Comment by NoahTheDuke** [comment]
Excellent, thank you!
